index.js

```diff
@@ -13,8 +13,11 @@ module.exports = function() {
 
   app.get('/', function(req, res) {
     var s = req.socket;
-    var msg = fmt('Hello, World! Request served from %s:%d\n',
-      s.localAddress, s.localPort);
+    var msg = fmt(
```

Comment:
Did `eslint` do this?

Reply:
I thought the eslint rules required it, but just checked, and it looks like it didn't. Not sure what happened, this was something I did a month ago. Anyhow, backed it out.